frontend/src/model/beta/Releases.tsx

```diff
@@ -28,13 +28,10 @@ export default function Releases({ model }: { model: ModelInterface }) {
     }
   }, [model, releases])
 
-  function handleDraftNewReleaseClose() {
-    setOpenDraftNewRelease(false)
-  }
-
   function handleDraftNewRelease() {
```

Comment:
Instead of using the `router.push()` function could we wrap the button in  `<Link />` to keep consistency with button navigations